### PR TITLE
Explicitly sets a timezone for querydsl-sql tests.

### LINKED ISSUE #2098: Tests fail when running in Australia/Sydney (& presumably others). We should set an explicit timezone in maven config.

### PR DESCRIPTION
**Pull request by karahatherly**
Fixes #2098 

Recreation of https://github.com/querydsl/querydsl/pull/2099 because I messed up on issue numbers.

### CODE REVIEW

**Comment by Shredder121** [comment]
Ah, I see you opened a new PR.
You can also edit the PR's description when you're the author.

But that's for the next time then.

File: querydsl-sql/pom.xml [hunk 1]
```diff
@@ -259,6 +259,7 @@
                 <value>target/derby.log</value>
             </property>
           </systemProperties>
+          <argLine>${surefire.argLine} -Duser.timezone=UTC</argLine>
```
Comment (Shredder121) [inline querydsl-sql/pom.xml]: You can remove this line entirely.

File: querydsl-sql/pom.xml [hunk 2]
```diff
@@ -21,7 +21,7 @@
       org.joda.time.*;version="[1.6,3)",
       ${osgi.import.package.root}
     </osgi.import.package>
-
+    <surefire.argLine/>
```
Comment (Shredder121) [inline querydsl-sql/pom.xml]: This line should be:
```xml
<argLine>${argLine} -Duser.timezone=UTC</argLine>
```

Comment (Shredder121) [inline querydsl-sql/pom.xml]: Hmm, I see that Maven complains about a recursive definition, I'll have to think about a solution.

**Comment by karahatherly** [comment]
@Shredder121 unfortunately I couldn't find a way to change the source branch (which contained the wrong issue number) for the PR in the github UI - figured recreating was harmless enough.

**Comment by Shredder121** [comment]
Yes, creating another PR is fine, but you can also push to the branch and the changes will reflect on here.
It's your fork and branch, so force pushing is also an option.

**Comment by Shredder121** [comment]
Looks good, sorry I didn't notice it was changed.

**Comment by timowest** [comment]
Looks good

**Comment by Shredder121** [comment]
👍 Now we just need to wait for some docker magic from @johnktims.
👀 
And then we have working tests again.

**Pull request merged**